holidays/countries/suriname.py

```diff
@@ -0,0 +1,261 @@
+        )
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_WORKDAY)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        dt = self._add_new_years_day(tr("Nieuwjaarsdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if (1981 <= self._year < 1993) or (2012 <= self._year < 2021):
+            # Day of Liberation and Renewal.
+            self._add_holiday_feb_25(tr("Dag van Bevrijding en Venieuwing"))
+
+        # Good Friday.
+        dt = self._add_good_friday(tr("Goede Vrijdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        # Easter Monday.
+        dt = self._add_easter_monday(tr("Tweede Paasdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year < 1976:
+            # Birthday of H.M. the Queen.
+            self._add_holiday_apr_30(tr("Verjaardag van H.M. de Koningin"))
+
+        # Labor Day.
+        dt = self._add_labor_day(tr("Dag van de Arbeid"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        # Day of Freedoms.
+        name = tr("Dag der Vrijheden") if self._year <= 2007 else tr("Keti Koti Dey")
+        dt = self._add_holiday_jul_1(name)
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year >= 2007:
+            # Indigenous People Day.
+            dt = self._add_holiday_aug_9(tr("Dag der Inheemsen"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 2012:
+            # Day of the Maroons.
+            dt = self._add_holiday_oct_10(tr("Dag der Marrons"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 1976:
+            # Republic Day.
+            name = "Dag van de Republiek" if self._year <= 2007 else tr("Onafhankelijkheidsdag")
+            dt = self._add_holiday_nov_25(tr(name))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        # Christmas Day.
+        dt = self._add_christmas_day(tr("Eerste Kerstdag"))
+        if self._year >= 2024:
+            self._add_observed(dt)
+
+        # Boxing Day.
+        dt = self._add_christmas_day_two(tr("Tweede Kerstdag"))
+        if self._year >= 2024:
+            self._add_observed(dt)
+
+        # Holi.
+        dt = self._add_holi(tr("Holi-dag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year >= 2012:
+            # Diwali.
+            dt = self._add_diwali(tr("Divali dag"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        # Eid al-Fitr.
+        for dt in self._add_eid_al_fitr_day(tr("Id-ul-Fitr dag")):
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 2012:
+            # Eid al-Adha.
+            for dt in self._add_eid_al_adha_day(tr("Id-ul-Adha dag")):
```

Comment:
"S.B. 2021 no. 27" names these holidays "Ied Ul Fitre" / "Ied Ul Adha", and "S.B. 2024 no. 167" - "Ied-Ul-Fitre" / "Ied-Ul-Adha"; maybe we should consider the latest change (2024)?